### PR TITLE
[Everhour] fix #13548 + modernize + add metadata/images,README

### LINKED ISSUE #13548: Everhour Time Tracking not loading tasks
### Extension

https://www.raycast.com/farisaziz12/everhour

### Raycast Version

1.79.0

### macOS Version

14.5 (23F79)

### Description

Everhour integration is not working: it's not loading tasks, so you can start/stop them. Here is an error screen: https://monosnap.com/direct/EmqET0Io0XBHb10qHn9JRDnF98m1TG. thx!

### Steps To Reproduce

1. start recent tasks everhour extension in Raycast, and you'll see the error https://monosnap.com/direct/EmqET0Io0XBHb10qHn9JRDnF98m1TG

### Current Behaviour

_No response_

### Expected Behaviour

_No response_

### PR DESCRIPTION
**Pull request by xmok**
## Description

- Fix Issue [#13548](https://github.com/raycast/extensions/issues/13548)
    - The issue would show up if a time entry did not have a task assigned
- Add icons to Actions
- Make `api` more TS friendly by removing use of `any`
- Add metadata images
- Add a README
~- commented out unused function but left it in case it is needed in future~
- removed redundant `keywords`

## Screencast

https://github.com/user-attachments/assets/15bc98c8-b065-4c14-ad32-c8fbbca51e43

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/everhour/src/api/index.ts [hunk 1]
```diff
@@ -16,27 +24,32 @@ const daysAgo = (days: number) => {
...
-  });
+// export const getCurrentUser = async () => {
+//   const response = await fetch("https://api.everhour.com/users/me", {
+//     headers,
+//   });
 
-  return (await response.json()) as any;
-};
+//   return (await response.json()) as any;
+// };
```
Comment (farisaziz12) [inline extensions/everhour/src/api/index.ts:33]: if not needed then happy to get rid of this code, lets not leave it commented out

File: extensions/everhour/src/views/RecentTaskList.tsx [hunk 2]
```diff
@@ -11,7 +11,10 @@ export function RecentTaskList() {
   const [isLoading, setIsLoading] = useState<boolean>(true);
 
   const refreshActiveTimer = async () => {
-    const toast = await showToast(ToastStyle.Animated, "Refreshing tasks");
+    const toast = await showToast({
+      style: Toast.Style.Animated,
+      title: "Refreshing tasks",
```
Comment (farisaziz12) [inline extensions/everhour/src/views/RecentTaskList.tsx:16]: Let's keep Capital Case for consistency like for all other toasts 

```suggestion
      title: "Refreshing Tasks",
```

File: extensions/everhour/src/views/RecentTaskList.tsx [hunk 3]
```diff
@@ -28,7 +31,10 @@ export function RecentTaskList() {
 
   useEffect(() => {
     async function fetch() {
-      const toast = await showToast(ToastStyle.Animated, "Fetching tasks");
+      const toast = await showToast({
+        style: Toast.Style.Animated,
+        title: "Fetching tasks",
```
Comment (farisaziz12) [inline extensions/everhour/src/views/RecentTaskList.tsx:36]: ditto ☝️

Comment (farisaziz12) [inline extensions/everhour/src/views/RecentTaskList.tsx:36]:
```suggestion
        title: "Fetching Tasks",
```

File: extensions/everhour/src/views/TaskList.tsx [hunk 4]
```diff
@@ -23,7 +23,10 @@ export function TaskList({
   const [isLoading, setIsLoading] = useState<boolean>(true);
 
   const refreshActiveTimer = async () => {
-    const toast = await showToast(ToastStyle.Animated, "Refreshing tasks");
+    const toast = await showToast({
+      style: Toast.Style.Animated,
+      title: "Refreshing tasks",
```
Comment (farisaziz12) [inline extensions/everhour/src/views/TaskList.tsx:28]:
```suggestion
      title: "Refreshing Tasks",
```

File: extensions/everhour/src/views/TaskList.tsx [hunk 5]
```diff
@@ -41,7 +44,10 @@ export function TaskList({
 
   useEffect(() => {
     async function fetch() {
-      const toast = await showToast(ToastStyle.Animated, "Fetching tasks");
+      const toast = await showToast({
+        style: Toast.Style.Animated,
+        title: "Fetching tasks",
```
Comment (farisaziz12) [inline extensions/everhour/src/views/TaskList.tsx:49]:
```suggestion
        title: "Fetching Tasks",
```

**Review by farisaziz12 — CHANGES_REQUESTED** [review]
Thanks for the contribution @xmok 🙌. Great stuff, just left a few minor comments

**Comment by xmok** [comment]
Appreciate the review @farisaziz12.
Removed the commented out code and made most toasts proper-cased.

**Review by farisaziz12 — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**